dev/release/binary-task.rb

```diff
@@ -1491,7 +1458,6 @@ def available_yum_targets
       ["almalinux", "8"],
       ["amazon-linux", "2"],
       ["centos", "7"],
-      ["centos", "8"],
```

Comment:
Am I missing something, or does this mean that CentOS 7 is still being supported?

Reply:
CentOS 7 has maintenance updates until 2024 https://wiki.centos.org/About/Product 
RHEL 9 is in beta https://developers.redhat.com/products/rhel/rhel9-beta

Reply:
Ah, amusing.